### PR TITLE
Drop timestamps from InstalledPackage model

### PR DESCRIPTION
**Pull request by lzap**
This pull request removes the "created_at", "deleted_at", and "updated_at" timestamps from the InstalledPackage model. The timestamps are no longer needed and removing them simplifies the code.

It introduces a manual migration package, let’s use this for all future migrations. The database also needs a massive review, schema is likely inconsistent.

### CODE REVIEW

**Comment by lzap** [comment]
What?!

```
  Error: cmd/migrate/manual/package.go:25:4: unreachable-code: unreachable code after this statement (revive)
  			log.Fatalf("Migration %s failed: %v", migration.Name, err)
  			^
```

File: pkg/models/commits.go [hunk 1]
```diff
@@ -66,7 +66,8 @@ type Package struct {
 
 // InstalledPackage represents installed packages a image has
 type InstalledPackage struct {
-	Model
+	ModelWithoutTimestamps
+	ID        uint     `gorm:"primarykey" json:"ID,omitempty"`
```
Comment (ezr-ondrej) [inline pkg/models/commits.go:70]: Now it's there twice, isn't it? :)

Comment (lzap) [reply pkg/models/commits.go:70]: Yeah.

Comment (lzap) [reply pkg/models/commits.go]: Interesting it worked, I think I know why :-) Nasty.

**Comment by ezr-ondrej** [comment]
Ugh it's used here: https://github.com/RedHatInsights/edge-api/blob/4761228b792651b4d22bb321fd7eee842bfc4009/pkg/clients/imagebuilder/client.go#L693 :face_exhaling: 
Easy fix there, but it worries me a little where else it's also used? :thinking:

**Comment by lzap** [comment]
Fixed this one. Don’t ask me, I did not write this project. 🤷🏻 Let’s merge and test the UI on stage.

**Review by ezr-ondrej — APPROVED** [review]
Nice :orange_heart:  LGTM :+1:

**Comment by ezr-ondrej** [comment]
I'll hold off merge for @loadtheaccumulator to have a chance to give it a peak :)

**Comment by ezr-ondrej** [comment]
/retest

Unrelated: `exists in quay.io/cloudservices/edge-api: authentication required`

**Comment by lzap** [comment]
@loadtheaccumulator please review, keep an eye on stage pod initialization to confirm it works once merged. Perhaps copy and store logs and paste them somewhere.

**Review by loadtheaccumulator — APPROVED** [review]
lgtm

**Pull request merged**